R/get_contrasts.R

```diff
@@ -288,7 +290,7 @@ get_contrasts <- function(model,
 
     # drop missing otherwise get_averages() fails when trying to take a simple mean
     idx_na <- !is.na(out$predicted_lo)
```

Comment:
Cool! We could also move `na.omit()` after the bayes stuff and only compute `idx_na` if we need to operate on `draws`.

Reply:
I guess this would have to be a "true" if-else, where `na.omit()` is also called in the bayes block because `out` is used there too.

Reply:
But I think `out` without the `NA`s is used a few lines after in `subset(out,...)`? So I guess we still need to use `na.omit()` before the bayes stuff

Reply:
Maybe I'm confusing several things, you can also commit directly to the branch if you prefer

Reply:
Done